packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -2103,7 +2103,9 @@ async function assertThemedProps(page: E2EPage, options: TestTarget): Promise<vo
   await page.mouse.reset();
   await page.waitForChanges();
 
+  // eslint-disable-next-line @stencil-community/strict-boolean-conditions
```

Comment:
Can you revert this before installing? It's only showing warnings and I've got a [PR to disable this rule](https://github.com/Esri/calcite-design-system/pull/9164/).